src/editors/containers/TextEditor/components/SelectImageModal/hooks.js

```diff
@@ -53,9 +54,11 @@ export const imgListHooks = ({
       onClick: () => setSelection(images[highlighted]),
     },
     galleryProps: {
+      galleryIsEmpty: Object.keys(images).length === 0,
```

Comment:
This is run once-- should it live inside a useffect, so it occurs on render?

Reply:
imgListHooks is called when Imghooks is called, which in theory should only be when the SelectImageModal renders or re-renders? Does that occur on uploading of an image? Just a little confused on how we can count on that logic.

Reply:
I believe gallery should be re-rendering every time the displayList or list of images changes... I understand the concern on the logic not running, but this has been working when I test it.